$OpenBSD: patch-ghc_mk,v 1.2 2010/06/15 15:34:03 kili Exp $

I don't see the point in not installing libraries that are required
by the build (and by several external haskell libraries we have or
will have in the ports tree).

--- ghc.mk.orig	Tue May 25 20:10:08 2010
+++ ghc.mk	Wed May 26 20:30:13 2010
@@ -844,7 +844,7 @@ INSTALLED_GHC_REAL=$(DESTDIR)$(bindir)/ghc.exe
 INSTALLED_GHC_PKG_REAL=$(DESTDIR)$(bindir)/ghc-pkg.exe
 endif
 
-INSTALLED_PACKAGES = $(filter-out haskeline mtl terminfo utf8-string,$(PACKAGES))
+INSTALLED_PACKAGES = $(PACKAGES)
 HIDDEN_PACKAGES = ghc-binary
 
 define set_INSTALL_DISTDIR
